rest_framework_simplejwt/serializers.py

```diff
@@ -105,25 +106,53 @@ class TokenRefreshSerializer(serializers.Serializer):
     token_class = RefreshToken
 
     default_error_messages = {
-        "no_active_account": _("No active account found for the given token.")
+        "no_active_account": _("No active account found for the given token."),
+        "password_changed": _("The user's password has been changed."),
     }
 
     def validate(self, attrs: dict[str, Any]) -> dict[str, str]:
         refresh = self.token_class(attrs["refresh"])
 
+        data = {"access": str(refresh.access_token)}
```

Comment:
shouldn't this be moved until after L136 so that we avoid computation if the token is revoked due to the password change?